src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculation = {
```

Comment:
Would be better to name this object as `calculator`

Reply:
👌 fixed